dashboard/db/migrate/20221003141047_update_course_version_index.rb

```diff
@@ -0,0 +1,11 @@
+class UpdateCourseVersionIndex < ActiveRecord::Migration[6.0]
+  def up
+    add_index :course_versions, [:course_offering_id, :key, :content_root_type], unique: true, name: 'index_course_versions_on_offering_id_and_key_and_type'
+    remove_index :course_versions, [:course_offering_id, :key]
+  end
+
+  def down
+    add_index :course_versions, [:course_offering_id, :key], unique: true
```

Comment:
Does the name get restored here automatically? Thinking of `index_course_versions_on_course_offering_id_and_key`

Reply:
Not sure it matters?!

Reply:
good question! yes the name does get restored here automatically, and yes it matters! 😁  

I had to specify the name on the other one because the name it wanted to generate by default was too long. this one uses the default, since it is short enough.

Reply:
Got it, thanks!